src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/services/CaseUpdater.java

```diff
@@ -44,6 +50,11 @@ public SuccessfulUpdateResponse update(CaseUpdateRequest caseUpdateRequest) {
             "Missing scanned documents in exception record"
         );
 
+        final List<String> warnings =
+            caseUpdateRequest.caseUpdateDetails == null
```

Comment:
are `caseUpdateDetails` not required?

Reply:
i would assume so too

Reply:
we are going to support case_update_details and exception_record for some time and case_update_details  is not sent by a functional test. I discussed this with Rafal and he agreed